src/renderer/api/kube-object.ts

```diff
@@ -115,12 +115,12 @@ export class KubeObject implements ItemObject {
     return KubeObject.stringifyLabels(this.metadata.labels);
   }
 
-  getAnnotations(): string[] {
+  getAnnotations(filter = false): string[] {
```

Comment:
Maybe it's better to make first argument `filter` as a callback: `filter: (label: string) => boolean` for more flexible filtering.

So instead of `...getAnnotations(true)` would be like:
```ts
...getAnnotations(label => resourceApplierApi.annotations.some(skipLabel => !label.startsWith(skipLabel)))
```

Reply:
It doesn't look good when multi places have logic like `...getAnnotations(label => resourceApplierApi.annotations.some(skipLabel => !label.startsWith(skipLabel)))`

Reply:
it's okay, they just should not repeating same inline filter and extracted to function in order to reuse.
Currently it's always bound to some dependent label list, but maybe it's fine for now.